docs/installation/configuration/cmake_options.rst

```diff
@@ -0,0 +1,173 @@
+        - ``OFF``
+    *   - :class:`INTERNAL_DEBUG`
+        - Activate developer debug messages. It is set to ``ON`` if :class:`EPROSIMA_BUILD` is ``ON``.
+        - ``OFF``
+
+Third-party libraries options
+-----------------------------
+
+Fast DDS relies on the `eProsima FastCDR <https://github.com/eProsima/Fast-CDR>`_ library for serialization
+mechanisms.
+Moreover, Fast DDS requires two internal third-party libraries for its proper operation: Asio and TinyXML2.
+Asio is a cross-platform C++ library for network and low-level I/O programming; while TinyXML2 parses the XML profile
+files, for later translation into C++ source code.
+These libraries can also be configured using Fast DDS CMake options.
+
+.. list-table::
+    :header-rows: 1
+
+    *   - Option
+        - Description
+        - Default
+    *   - :class:`THIRDPARTY`
+        - Activates the use of Fast CDR and the internal third-party libraries. It is set to ``ON`` |br|
+          if EPROSIMA_BUILD is ``ON``.
+        - ``OFF``
+    *   - :class:`THIRDPARTY_UPDATE`
+        - Activates the automatic update of Fast CDR and the internal third-party libraries.
+        - ``ON``
+    *   - :class:`THIRDPARTY_fastcdr`
+        - Activates the use of Fast CDR library.
+        - ``OFF``
+    *   - :class:`THIRDPARTY_Asio`
+        - Activates the use of Asio internal third-party library.
+        - ``OFF``
+    *   - :class:`THIRDPARTY_TinyXML2`
+        - Activates the use of TinyXML2 internal third-party library.
+        - ``OFF``
+    *   - :class:`THIRDPARTY_android-ifaddrs`
+        - Activates the use of android-ifaddrs, an implementation of :func:`getifaddrs` for Android. |br|
+          Only if :class:`ANDROID` is set to ``ON``.
+        - ``OFF``
+
+Test options
+------------
+
+*eProsima Fast DDS* comes with a full set of tests to check the performance of the library.
+These tests are divided according to the layer being tested; thus there are DDS-layer tests and RTPS-layer tests.
+The building and execution of these tests is specified by the Fast DDS CMake options shown in the table below.
+
+.. list-table::
+    :header-rows: 1
+
+    *   - Option
+        - Description
+        - Default
+    *   - :class:`FASTRTPS_API_TESTS`
+        - Enables the execution of black-box tests for Fast DDS using the Fast DDS RTPS-layer API.
+        - ``OFF``
+    *   - :class:`FASTDDS_PIM_API_TESTS`
+        - Enables the execution of black-box tests for Fast DDS using the Fast DDS DDS-layer API.
+        - ``OFF``
+
+Fast DDS tests are implemented using the GoogleTest framework.
+To run the tests it is required to enable this framework using the :class:`GTEST_INDIVIDUAL` option.
+The configuration of the installation directories is also allowed as indicated in the following table.
+
+.. list-table::
+    :header-rows: 1
+
+    *   - Option
+        - Description
+        - Default
+    *   - :class:`GTEST_INDIVIDUAL`
+        - Activate the execution of GoogleTest tests.
```

Comment:
No exactly. It activates the individual building of the tests. However, the test are compiled if `EPROSIMA_BUILD=ON` (not 100% sure that's the correct option). In that case, if `GTEST_INDIVIDUAL=OFF`, then when you run the tests, they are one huge test, instead of 1000+ individual ones.